app/web/src/newhotness/explore_grid/ExploreGridTile.vue

```diff
@@ -23,9 +23,9 @@
     >
       <Icon :name="getAssetIcon(component.schemaCategory)" size="lg" />
       <h2>
-        <TruncateWithTooltip class="pb-xs">{{
-          component.name
-        }}</TruncateWithTooltip>
+        <TruncateWithTooltip class="pb-xs"
+          >name: {{ component.name }}</TruncateWithTooltip
```

Comment:
is this right? having `name: ` in the text?

Reply:
ok @jobelenus removed